contracts/tools/testing/slither/Slither.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-FileCopyrightText: 2024 IEXEC BLOCKCHAIN TECH <contact@iex.ec>
+// SPDX-License-Identifier: Apache-2.0
+
+pragma solidity ^0.8.0;
+
+import {IexecPocoAccessorsDelegate} from "../../../modules/delegates/IexecPocoAccessorsDelegate.sol";
+import {IexecPoco1Delegate} from "../../../modules/delegates/IexecPoco1Delegate.sol";
+import {IexecPoco2Delegate} from "../../../modules/delegates/IexecPoco2Delegate.sol";
+
+/**
+ * @notice This contract is dedicated to slither analysis.
```

Comment:
Can you add here the reason why it was created, please.

Reply:
Done see : [update notices](https://github.com/iExecBlockchainComputing/PoCo/pull/87/commits/d2594ba5fa7c3653c5fe101697cf05a8125e3293)

Reply:
Thanks!